src/content/app/genome-browser/components/browser-sidebar-modal/BrowserSidebarModal.tsx

```diff
@@ -27,6 +27,8 @@ import { closeDrawer } from 'src/content/app/genome-browser/state/drawer/drawerS
 
 import SidebarModal from 'src/shared/components/layout/sidebar-modal/SidebarModal';
 
+import navigateModalStyles from './modal-views/navigate-modal/NavigateModal.scss';
```

Comment:
Mmm, the grey background feels like it belongs in BrowserSidebarModal's own scss file. What if any modal other than the navigation one wants to be grey?

Reply:
Yes that's a good point. However, isn't there a possibility grey background can be used in other apps as well? So should it go in SidebarModal.scss?

Reply:
If it goes there, then the way to invoke the grey background would be to add something like a `theme` prop to SidebarModal component. I am trying to think if there are any downsides to doing so, but can't think of any. Unless Andrea gets super inventive with the modals.

Reply:
✅